mm/kfence.c

```diff
@@ -57,6 +57,19 @@ struct kfence_freelist_t {
 	struct list_head list;
 	void *obj;
 };
+/*
+ * kfence_freelist_t is a wrapper around kfence page pointers that allows
+ * chaining them.
+ * |kfence_freelist| is a FIFO queue of non-allocated pages, |kfence_recycle| is
```

Comment:
The kernel uses '@' to refer to variables in documentation. E.g. "@kfence_freelist".

Reply:
I thought it's for function arguments/struct members only. I don't have a strong preference though.
Fixed.